backend/common/src/byte_size.rs

```diff
@@ -0,0 +1,107 @@
+// Source code for the Substrate Telemetry Server.
+// Copyright (C) 2021 Parity Technologies (UK) Ltd.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use anyhow::{anyhow, Error};
+
+#[derive(Copy, Clone, Debug)]
+pub struct ByteSize(usize);
+
+impl ByteSize {
+    pub fn new(bytes: usize) -> ByteSize {
+        ByteSize(bytes)
+    }
+    pub fn into_bytes(self) -> usize {
```

Comment:
I find `into_bytes()` a little confusing given how it's used in the Rust ecosystem.

When I see `into_bytes` I assume you will return a `vec/slice/array` of the actual bytes.

I think `len` or `size` would be more clear in this context.

Reply:
Yeah, `into_bytes` definitely isn't the best choice of name (nor is `ByteSize` really)! `byte_size` works :)